cloud/billing/pricing.go

```diff
@@ -40,7 +40,10 @@ var awsRegionNameToIDMap = map[string]string{
 	"EU (Ireland)":               "eu-west-1",
 	"EU (London)":                "eu-west-2",
 	"EU (Paris)":                 "eu-west-3",
+	"EU (Stockholm)":             "eu-north-1",
 	"South America (Sao Paulo)":  "sa-east-1",
+	"AWS GovCloud (US-East)":     "unknown",
+	"AWS GovCloud (US-West)":     "unknown",
```

Comment:
There is no west  gov cloud, just US and US-East:


https://docs.aws.amazon.com/general/latest/gr/rande.html

Reply:
Yeah, I saw that too, but from some reason it errored out when I left out `US-West` or just `US`. Not sure why!

Figured I would just keep them both there if that's not an issue?

Will update the values though, thanks!

Reply:
that's completely a fair approach, a second alias for west-gov is probably fine.